docs/spec_draft.md

```diff
@@ -2234,6 +2235,40 @@ hidden state.
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.round_nearest_even
+
+### Semantics
+
+Performs element-wise rounding towards the nearest integer, breaking ties
+towards the even integer, on the `operand` tensor and produces a `result`
+tensor.
```

Comment:
Let's also add a second sentence: "Implements the `roundToIntegralTiesToEven` operation from the IEEE-754 specification".